docs/source/pages/Configuration.rst

```diff
@@ -111,26 +109,24 @@ Interfaces
 
 Interfaces are all defined in **user/config.json** in the **services** section.
 
-More details details on how to setup OctoBot's interfaces in the Interfaces section.
-
 Trading and Risk parameter
 --------------------------
 
 OctoBot can process two types of trading:
 
 
 * Real trading using your exchanges' portfolio.
-  `Here are the details on how to setup a trader. <Trader.html>`_
+  `Here are the details on how to setup a trader. <https://drakkar-software.github.io/OctoBot/pages/Trader.html>`_
 * Simulated trading using any imaginary portfolio.
-  `Here are the details on how to setup a trader simulator. <Simulator.html>`_
+  `Here are the details on how to setup a trader simulator. <https://drakkar-software.github.io/OctoBot/pages/Simulator.html>`_
 
 
 .. image:: https://raw.githubusercontent.com/Drakkar-Software/OctoBot/assets/wiki_resources/trading.jpg
    :target: https://raw.githubusercontent.com/Drakkar-Software/OctoBot/assets/wiki_resources/trading.jpg
    :alt: trading
 
 
-Any type of trading has its **risk** parameter. It is a parameter defining the behavior of the trader, similarly to a real human trader. `This risk parameter is described here <Trader.html#risk>`_
+Any type of trading has its **risk** parameter. It is a parameter defining the behavior of the trader, similarly to a real human trader. `This **risk** parameter is described here <https://drakkar-software.github.io/OctoBot/pages/Trader.html#risk>`_
```

Comment:
is it not working on your setup ? it works on mine and on deployed pages. I think we should avoid using the full URL to be able to change it if necessary without having to update each and every link

Reply:
We should also avoid using hardcoded html file name